File: 프로그래머스/0/120585. 머쓱이보다 키 큰 사람/머쓱이보다 키 큰 사람.py
def solution(array, height):
    # 1
    # count = 0
    # for i in array :
    #     if i > height :
    #         count += 1
    # return count      
    
    # 2
    # array.append(height)
    # return sorted(array, reverse=True).index(height)
   
    # 3
    return len(list(filter(lambda x:x > height, array)))

File: 프로그래머스/0/120585. 머쓱이보다 키 큰 사람/머쓱이보다 키 큰 사람.js
function solution(array, height) {
    // 1
    // var answer = 0;
    // for(let i = 0; i < array.length; i++) {
    //     if(array[i] > height) {
    //         answer++;
    //     }
    // }
    // return answer;
    
    // 2
    // let count = 0;
    // for (i of array) {
    //     if (i > height) {
    //         count += 1
    //     }
    // }
    // return count
    
    // 3
    return array.filter(v => v > height).length;
}